### ISSUE TITLE
Wrong arguments starting fast-vladmantic-webui

### ISSUE THREAD

**Issue by BinaryQuantumSoul** [Linaqruf/sd-notebook-collection#20]
Hello, using the default [fast-vladmantic-webui](https://colab.research.google.com/drive/126cDNwHfifCyUpCCQF9IHpEdiXRfHrLN),
 There is an error when starting webui:
 ```
error: unrecognized arguments: --enable-insecure --no-download-sd-model
```

I fixed it replacing `--no-download-sd-model` with `--no-download` and removing line `enable-insecure : True` in the config json. I don't know if it is good practice though.

**Comment by LukeOvermind** [comment]
i get 

error: unrecognized arguments: --enable-insecure --no-download-sd-model

**Comment by BinaryQuantumSoul** [comment]
> error: unrecognized arguments: --enable-insecure --no-download-sd-model

Same than me then

**Comment by Linaqruf** [comment]
Thank you for the feedback, and sorry for my late reply. 
I rarely check GitHub lately.

I renamed `--enable-insecure` to `--insecure` and `--no-download-sd-model` to `--no-download`

it should work now.

Reference: https://github.com/vladmandic/automatic/blob/master/modules/cmd_args.py

**Comment by BinaryQuantumSoul** [comment]
> Thank you for the feedback, and sorry for my late reply. 
> I rarely check GitHub lately.
> 
> I renamed `--enable-insecure` to `--insecure` and `--no-download-sd-model` to `--no-download`
> 
> it should work now.
> 
> Reference: https://github.com/vladmandic/automatic/blob/master/modules/cmd_args.py

Thank you.

Any chance you could add the notebook to the github so we can PR in those cases ?

**Comment by Linaqruf** [comment]
Yes, I already add the notebook to github, but probably I won't update it more often.
https://github.com/Linaqruf/vladmandic-webui-colab
